src/components/Explore/TokenDetail.tsx

```diff
@@ -195,7 +196,7 @@ export default function LoadedTokenDetail({ address }: { address: string }) {
             {tokenName} <TokenSymbol>{tokenSymbol}</TokenSymbol>
           </TokenNameCell>
           <TokenActions>
-            <Share size={18} />
+            {tokenName && tokenSymbol && <ShareButton tokenName={tokenName} tokenSymbol={tokenSymbol} />}
```

Comment:
multiple truthy values in a template can lead to unexpected render behavior--should compress this statement into a single boolean for the template render decision